frontend/src/components/tabs/StaticNodeDetailsV2.test.tsx

```diff
@@ -119,6 +135,7 @@ describe('StaticNodeDetailsV2', () => {
 
     screen.getByText('Command');
     expect(screen.getAllByText('sh').length).toEqual(2);
+    // See Line 83
```

Comment:
Don't use line number in comment. Line number can change easily and we lost track of it.